.github/workflows/win-gpu-x64-build.yml

```diff
@@ -35,17 +33,26 @@ jobs:
       run: |
         azcopy.exe cp --recursive "https://lotusscus.blob.core.windows.net/models/cuda_sdk/v${{ env.cuda_version }}" ${{ env.cuda_dir}}
 
-    - name: Download OnnxRuntime
+    - name : Install jq and nuget
       run: |
-        Invoke-WebRequest -Uri $env:ort_url -OutFile $env:ort_zip
+        choco install -y jq nuget.commandline curl
```

Comment:
Use the "UseDotnet" task instead.

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-dotnet-v2?view=azure-pipelines

The machine pool is shared by multiple project. Please avoid making global changes.